src/wallet/mod.rs

```diff
@@ -619,17 +624,8 @@ impl Wallet {
         with_connection(&self.db_connection.lock())
     }
 
-    pub async fn get_sidechains(&self) -> Result<Vec<Sidechain>> {
-        let sidechains = self
-            .validator
-            .get_sidechains()?
-            .into_iter()
-            .map(|sidechain| Sidechain {
-                sidechain_number: sidechain.sidechain_number,
-                data: sidechain.data,
-            })
-            .collect();
-        Ok(sidechains)
+    pub async fn get_active_sidechains(&self) -> Result<Vec<Sidechain>> {
+        self.validator.get_active_sidechains()
     }
```

Comment:
Can delete this method IMO. It's not used outside of this file, and that place can just call `self.validator.get_active_sidechains`